CONTRIBUTING.md

```diff
@@ -114,6 +114,30 @@ If for any reason there is an update to the build tools, you will then need to f
 3. Invoke `dotnet test eng\service.proj --filter TestCategory!=Live`
    <br/><br/>
 
+### Live testing
+
+Live tests assume a live resource has been created and appropriate environment
+variables have been set for the test process. To automate setting up live
+resources we use created a script called `New-TestResources.ps1` that deploys
+resources for a given service.
+
+To see what resources will be deployed for a live service, check the
+`test-resources.json` ARM template files in the service you wish to deploy for
+testing.
+
+To deploy live resources for testing use the steps documented in [`Example 1 of New-TestResources.ps1`](eng/common/TestResources/New-TestResources.ps1.md#example-1)
+to set up a service principal and deploy live testing resources.
+
+To run live tests after deploying live resources
+
+1. Open VS 2019 Command Prompt
+2. Navigate to service directory e.g. _"sdk\keyvault"_
+3. Invoke `dotnet test` _(note no `--filter` parameter)_
```

Comment:
Should they be used to specifying a `--filter` parameter? I never have. Seems like one of those things you'll mention that no one would do, but now they might because you made them aware. I'd just leave the parenthetical off.